receive_data/receive_data.ino

```diff
@@ -0,0 +1,56 @@
+/*
+ * Color By Number/Neighbor
+ * 
+ * An example showing how to count the number of neighbors.
+ * 
+ * Each Blink displays a color based on the number of neighbors around it.
+ *  
+ */
+
+// color by number of neighbors
+Color colors[] = {  
+  dim(WHITE,40),   // 0 neighbors
+  RED,            // 1 neighbors
+  YELLOW,         // 2 neighbors
+  GREEN,          // 3 neighbors
+  CYAN,           // 4 neighbors
+  BLUE,           // 5 neighbors
+  MAGENTA,        // 6 neighbors
+}; 
+
+
+
+void setup() {
+  // Blank
+}
+
+
+void loop() {
+
+  // count neighbors we have right now
+  int numNeighbors = 0;
+ 
+  FOREACH_FACE(f) {
+
+    if ( !isValueReceivedOnFaceExpired( f ) ) {   // Have we seen an neighbor on this face recently?
+          if ((int)getLastValueReceivedOnFace(f) == 10){
+            numNeighbors = 1;
+          }
+          if ((int)getLastValueReceivedOnFace(f) == 30){
+            numNeighbors = 2;
+          }
+          if ((int)getLastValueReceivedOnFace(f) == 60){
+            numNeighbors = 3;

```

Comment:
I think we could completely remove the logic about neighbors and just call `setColor(CYAN)` for example